config/vufind/contentsecuritypolicy.ini

```diff
@@ -25,10 +25,16 @@ use_nonce = true
 ; See also the VuFind wiki for additional recommendations and tools:
 ; https://vufind.org/wiki/administration:security:content_security_policy
 [Directives]
-default-src[] = "'none'"
-; 'strict-dynamic' would allow any trusted script to load other scripts. However, it
-; is currently not supported by Safari.
+; default of 'self' with 'none' on child, object, prefetch allows SVG requests.
+default-src[] = "'self'"
+child-src[] = "'none'"
+object-src[] = "'none'"
+prefetch-src[] = "'none'"
+; 'strict-dynamic' allows any trusted script to load other scripts with a hash.
+; Recent versions of Safari do not support this feature, so it is disabled by default.
```

Comment:
"Recent"?

Reply:
Do we have a specific version number we can use for cut-off, like "Versions before X (released in YEAR) do not..."